plugins/org.yakindu.sct.doc.user/src/user-guide/simulating_statecharts.textile

```diff
@@ -117,52 +116,58 @@ p=. The _SC Simulation_ perspective
 
 h2(#simu_the_simulation_view). The _Simulation_ view
 
-The _Simulation_ view is used to manually raise events and to inspect and modify variables of a running simulation. By default that view is located on the right-hand side of the "_SC Simulation_":#simu_the_sc_simulation_perspective perspective, see "figure &quot;Simulation view&quot;":#simu_fig_simulation_view for an example.
+The _Simulation_ view is used to manually raise events and to inspect and modify variables of a running simulation. By default, that view is located on the right-hand side of the "_SC Simulation_":#simu_the_sc_simulation_perspective perspective, see "figure &quot;Simulation view&quot;":#simu_fig_simulation_view for an example.
 
-The _Simulation_ view groups events and variables by their _interfaces_. The unnamed interface appears as _default_ in the interface list. Click on the small triangle left from a interface's name to show or hide the interface's contents, i.e., events and variables.
+The _Simulation_ view groups events and variables by their _interfaces_. The unnamed interface appears as _default_ in the interface list. Click on the small triangle left from an interface's name to show or hide that interface's contents, i.e., events and variables.
 
-bq.. *Please note:*
+The simulation view also displays _time events_, provided the statechart uses time constructs like _after_ or _every_. Press or release the _"show time events"_ button !(inlinemediaobject)images/symbol_event_time_016.png(symbol: "show time events")! to toggle between displaying and not displaying time events. You can click on a time event to raise it, i.e., you don't have to wait for 24 hours to elapse in real time until the _"after 24 * 60 * 60 s"_ transition fires.
 
-Depending on your screen resolution and font size settings, you might not be able to spot the _Simulation_ view by its name, because the tab containing it is quite narrow and might not provide enough space for displaying the title. Hover over the tabs to reveal their respective titles in a popup window.
+You can have multiple simulation instances running at the same time. They may simulate the same or different statecharts. Use the simulation view's drop-down menu to switch between simulation instances.
 
-"Figure &quot;The SC Simulation perspective&quot;":#simu_fig_the_sc_simulation_perspective is demonstrating this: The user has hovered the mouse pointer over a tab that is just displaying the starting letter 'S' of its title. However, a popup window right besides the pointer is showing the tab's full title "Simulation".
+A digital clock right from the drop-down menu displays the time spent in the simulation. It pauses while the simulation is suspended.
```

Comment:
I think this sounds like the clock would display the time the user has spent in the simulation, which is not quite the point. Instead, it displays the "virtual time" that has passed from the statechart's point of view - the statechart's time could technically very well run on double or half the normal speed, which would affect this clock.